src/bin/Makefile

```diff
@@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
         show-alignments compile-questions cluster-phones \
         compute-wer compute-wer-bootci make-h-transducer \
```

Comment:
Yes, please.


On Tue, Mar 22, 2016 at 5:50 PM, Kirill Katsnelson <notifications@github.com
> wrote:

> In src/bin/Makefile
> <https://github.com/kaldi-asr/kaldi/pull/579#discussion_r57075840>:
>
> > @@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
> >          show-alignments compile-questions cluster-phones \
> >          compute-wer compute-wer-bootci make-h-transducer \
>
> Sure. It also exists in the const-arpa compiler--do you want to remove it
> from that program too?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/579/files/ee02fd543e4531aa003cbf296fd6ba50f7d7ea12#r57075840>
>